crates/pop-parachains/src/utils/helpers.rs

```diff
@@ -24,6 +25,57 @@ pub(crate) fn sanitize(target: &Path) -> Result<(), Error> {
 	Ok(())
 }
 
+pub async fn clear_dmpq(
+	client: OnlineClient<PolkadotConfig>,
+	para_ids: &[u32],
+) -> Result<(), Box<dyn std::error::Error>> {
+	use subxt_signer::sr25519::dev;
+
+	#[subxt::subxt(runtime_metadata_path = "./src/utils/artifacts/paseo-local.scale")]
```

Comment:
This file is rather large. I believe there is a way to filter the metadata which is included based on only those explicit pallets required. Can we limit it to the system pallet only?

Reply:
Ideally we could generate the interface from the given client on the fly ? I don't know how feasible is that

Reply:
Will see how small the resulting metadata is with only the required pallet first, as generating it on every run or caching it brings additional work/complexities.

Reply:
FYI - reduced from 308KB to 48 KB with:

`subxt metadata --url wss://rpc.ibp.network/paseo --pallets System,Sudo > paseo.scale`

Reply:
There is a problem with the filtering sadly: https://github.com/paritytech/subxt/issues/1659